DESCRIPTION

```diff
@@ -3,16 +3,20 @@ Type: Package
 Title: Calculate and Visualize Synergy Scores for Drug Combinations
 Version: 1.3.0
 Date: 2016-04-15
-Author: Liye He <liye.he@helsinki.fi>, Jing Tang <jing.tang@helsinki.fi>
+Author: Liye He <liye.he@helsinki.fi>, Jing Tang <jing.tang@helsinki.fi>,
+        David L. Roxe <dlroxe@google.com>
 Maintainer: Liye He <liye.he@helsinki.fi>
-Imports: drc (>= 2.5-12), reshape2 (>= 1.4.1), SpatialExtremes (>= 2.0-2),
+Imports: drc (>= 2.5-12), reshape2 (>= 1.4.1), SpatialExtremes,
         ggplot2 (>= 2.1.0), gridBase (>= 0.4-7), grid (>= 3.2.4),
         lattice (>= 0.20-33), gplots (>= 3.0.0), nleqslv(>= 3.0), stats
         (>= 3.3.0), graphics (>= 3.3.0), grDevices (>= 3.3.0)
-Description: Efficient implementations for all the popular synergy scoring models for drug combinations, including HSA, Loewe, Bliss and ZIP and visualization of the synergy scores as either a two-dimensional or a three-dimensional interaction surface over the dose matrix.
+Description: Efficient implementations for all the popular synergy scoring
+             models for drug combinations, including HSA, Loewe, Bliss and ZIP
+             and visualization of the synergy scores as either a two-dimensional
+             or a three-dimensional interaction surface over the dose matrix.
 License: Artistic License 2.0
```

Comment:
I showed them the DESCRIPTION file as it appeared in my proposed changes.  They saw the license and the addition of my @google.com address, and approved it.  So this is fine as-is.

Reply:
My colleague recommended MIT license since she also used MIT license in her R package. Therefore, I changed to the license to MIT license. Please check your employer is okay with MIT license.

Reply:
I asked.  Basically, it's fine.  It's perhaps easier if you quickly check in a change to the DESCRIPTION file to indicate the MIT license.  That way, there's no confusion when I check in -- I'm checking in to an MIT-licensed project (which is OK with them, still providing that my @google.com email address appears in the authors list where the license is specified).

Reply:
Slight terminology flub on my part:  It might be easier for you to push your own change to the DESCRIPTION file before accepting my pull request.  (At work, I'm used to being the one to submit the change after my reviewer says "OK", but I think GitHub works the other way.)